modules/parent-join/src/main/java/org/elasticsearch/join/mapper/MetaJoinFieldMapper.java

```diff
@@ -40,8 +40,8 @@
  * This class is also used to quickly retrieve the parent-join field defined in a mapping without
  * specifying the name of the field.
  */
-class MetaJoinFieldMapper extends FieldMapper {
-    static final String NAME = "_parent_join";
+public class MetaJoinFieldMapper extends FieldMapper {
+    public static final String NAME = "_parent_join";
```

Comment:
Ok now I understand why you want to have a single package ;)
Can you use "_parent_join" directly in the test and we'll revisit this when we move to a single package ?